LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright © 2011 Mozilla Foundation
```

Comment:
I updated all the source code headers to contain ISC license instead of MPL 2.0 header.